### PR TITLE
tests: change the packages for the security status test (SC-1353)

### PR DESCRIPTION
**Pull request by renanrodrigo**
To test the full output of security-status, we install packages to count them as third-party and unknown.
We install `code` to be the third party option, and `gh` to be the unknown package. The reason for that is that the `code` deb sets up a parts file with a source entry (thus being recognized as third-party) and `gh ` does not.

However, `gh` just landed on `universe` from `kinetic` on. So, when testing for the release (when u-a-t is installed from `-proposed`), there is no unknown package anymore :|

[EDITED] 
~The """"fix"""" here is to change the `gh` package to any other which is not in the archive. The chosen one is `surfshark-release` just because I could not find a `.deb` in the wild which wouldn't setup a repository. This one does, but we remove the parts file to make sure it is counted as unknown.~

The better and actual fix suggested by @orndorffgrant here is to install dummy packages that we created instead.
Example packages were created and hosted appropriately in a S3 bucket, and the approach works like a charm.

F̶e̶e̶l̶ ̶f̶r̶e̶e̶ ̶t̶o̶ ̶h̶a̶t̶e̶ ̶t̶h̶i̶s̶ ̶P̶R̶.̶ 
It is mandatory to love this PR now.

## Test Steps
Run the Kinetic security-status test, installing u-a-t from any source not local/prebuilt, and see it fail. Apply this change and see it passing.

## Desired commit type
 - [ ] Squash and merge: Using the "Proposed Commit Message"
 - [ ] Create a merge commit and use "Proposed Commit Message"
 - [x] Rebase and merge, no merge commit, rely only on existing commit messages

## Checklist
 - [ ] I have updated or added any unit tests accordingly
 - [x] I have updated or added any integration tests accordingly
 - [ ] I have updated or added any documentation accordingly

## Does this PR require extra reviews?
 - [ ] Yes
 - [x] No

### CODE REVIEW

**Review by orndorffgrant — APPROVED** [review]
Approving with the request that we change how this test works in the future.
Ideally we wouldn't be relying on these randomly picked third party debs.
We should build our own minimal debs specifically for this test.

**Comment by renanrodrigo** [comment]
@orndorffgrant the only thing now is that I am hosting those packages using my company credentials - which is fine for now, but in the future we should migrate those to some team-owned bucket.
If you think this is acceptable for now, then it is done. If you think it is harmful for me to host those, we can discuss and migrate, and then update this PR to point to the appropriate URLs before merging.

**Review by orndorffgrant — APPROVED** [review]
Created https://warthogs.atlassian.net/browse/SC-1362 to move them to team owned infra

Feel free to merge after resolving conflicts

**Pull request merged**
